docs/validators/stale.rst

```diff
@@ -4,13 +4,17 @@ Stale
 ::
 
     - do: stale
-    days: 20 # number of days ago.
-    type: pull_request, issues # what items to search for.
+      days: 20 # number of days ago.
+      type: pull_request, issues # what items to search for.
+      limit: # Optional, run only if the specific options are met
```

Comment:
TOL: This configuration type might be something we want add to another validators and actions in the future.

Reply:
hmm.. in that case, maybe we should call it `time_limit` so it can be reusable in all actions and validators. i'll fix it

Reply:
to close the loop here, i ended up name it `time_constraint` instead